src/secp256k1.c

```diff
@@ -4,6 +4,8 @@
  * file COPYING or https://www.opensource.org/licenses/mit-license.php.*
  ***********************************************************************/
 
+#define SECP256K1_BUILD
```

Comment:
Shouldn't this be wrapped in `#ifndef SECP256k1_BUILD`?

Reply:
I don't see why-- if it was you wouldn't have spotted that your rust build was incorrectly setting it now that it shouldn't be set that way anymore.

Reply:
I just thought it was common practice to wrap defines in ifndef in the case that the define acts as a flag and doesn't carry any additional value.
(I don't think there was any harm in the fact that we kept passing `-DSECP256K1_BUILD` to the compiler, as we don't really use "headers" in rust)